src/main/java/se/knowit/bookitevent/kafka/producer/KafkaEventProducerServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package se.knowit.bookitevent.kafka.producer;
+
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.support.SendResult;
+import org.springframework.util.concurrent.ListenableFuture;
+import org.springframework.util.concurrent.ListenableFutureCallback;
+import se.knowit.bookitevent.dto.EventDTO;
+
+public class KafkaEventProducerServiceImpl implements KafkaProducerService<EventDTO> {
+
+    private KafkaTemplate<String, EventDTO> eventTemplate;
+
+    public KafkaEventProducerServiceImpl(KafkaTemplate<String, EventDTO> eventTemplate) {
+        this.eventTemplate = eventTemplate;
+    }
+
+    @Override
+    public void sendMessage(final String topic, final EventDTO event) {
+        ListenableFuture<SendResult<String, EventDTO>> future = eventTemplate.send(topic, event);
```

Comment:
We need to use a key when we send so we can get the compacted topics to work for us